charts/jenkins/templates/_helpers.tpl

```diff
@@ -109,6 +109,9 @@ jenkins:
     enabled: true
   mode: {{ .Values.controller.executorMode }}
   numExecutors: {{ .Values.controller.numExecutors }}
+  {{- if .Values.controller.customJenkinsLabels }}
```

Comment:
please add a test for this

Reply:
Sorry, went out of town and this PR completely slipped my mind. I just added in a single test ensuring that it minimally starts off as an empty string, is that what you had in mind?